packages/idos-sdk-js/src/lib/enclave-providers/metamask-snap-enclave.ts

```diff
@@ -93,4 +93,8 @@ export class MetaMaskSnapEnclave implements EnclaveProvider {
 
     return Uint8Array.from(Object.values(decrypted));
   }
+
+  async comparePublicKeys(encryptionPublicKey: string, idOSPublicKey: string) {
+    return this.invokeSnap("comparePublicKeys", { encryptionPublicKey, idOSPublicKey });
+  }
```

Comment:
I'd maybe abstain from adding this method at all until we start using it.

But now I'm curious: what prompted you to add it? :thinking: Was is just to have _something_ in the snap enclave, or is there maybe something I'm not seeing?